Sources/SwiftInspectorVisitors/Tests/ExtensionVisitorSpec.swift

```diff
@@ -110,6 +110,23 @@ final class ExtensionVisitorSpec: QuickSpec {
             expect(extensionInfo?.inheritsFromTypes.map { $0.asSource }) == ["Foo", "Bar"]
           }
         }
+
+        context("with computed properties") {
+          it("finds the inheritance types") {
```

Comment:
copy/paste? I don't understand this `it` since nothing in this test has to do with inheritance

Reply:
copy/paste!

Reply:
https://github.com/fdiaz/SwiftInspector/pull/100/commits/4f14f652851b23219a06e580d10e013a49c1a494